djangodenmark/apps/newsletter/views.py

```diff
@@ -0,0 +1,31 @@
+from django.urls.base import reverse
+from django.views.generic.base import RedirectView
+from django_consent import models
+from django_consent.views import ConsentConfirmationSentView
+from django_consent.views import ConsentCreateView
+
+
+class Index(RedirectView):
+
+    permanent = False
+
+    def get_redirect_url(self, *args, **kwargs):
+        source = models.ConsentSource.objects.all().first()
+        if not source:
+            source = models.ConsentSource.objects.create(
+                source_name="Auto-created default newsletter"
+            )
+        return reverse("newsletter:signup", kwargs={"source_id": source.id})
```

Comment:
This is the really sloppy part. This should be a proper data migration. The /newsletter/ redirect is perhaps useful, but more likely completely useless :)